docs/tasks/administer-cluster/cilium-network-policy.md

```diff
@@ -67,7 +68,7 @@ cilium          1         1         1         <none>          2m
 There are two main components to be aware of:
 
 - One `cilium` Pod runs on each node in your cluster and enforces network policy on the traffic to/from Pods on that node using Linux BPF.
-- For production deployments, Cilium should leverage the key-value store cluster (e.g., etcd) used by Kubernetes, which typically runs on the Kubernetes master nodes.   The [Cilium Administrator Guide](http://cilium.readthedocs.io/en/latest/admin/) includes an example DaemonSet which can be customized to point to this key-value store cluster.   The simple ''all-in-one'' DaemonSet for minikube requires no such configuration because it automatically deploys a `cilium-consul` Pod to provide a key-value store.
+- For production deployments, Cilium should leverage the key-value store cluster (e.g., etcd) used by Kubernetes, which typically runs on the Kubernetes master nodes.   The [Cilium Administrator Guide](https://cilium.readthedocs.io/en/latest/install/#kubernetes-installation-guide) includes an example DaemonSet which can be customized to point to this key-value store cluster.   The simple ''all-in-one'' DaemonSet for minikube requires no such configuration because it automatically connects to the minikube's etcd instance.
```

Comment:
if the admin guide is gone, we should rename the link to installation guide.

Reply:
use single space between sentences.

Reply:
Addressed